addon/utils/text.js

```diff
@@ -71,8 +72,9 @@ export function trimDataToFit (text, width, font) {
  * Note: expects data-text attribute to be on element with full text
  *
  * @param {HTMLElement} element - HTML element to trim data within
+ * @param {jQuery} [$] - jQuery selector, can be stubbed for testing
```

Comment:
Unfortunately, for reasons unknown, setting the default for `$` as `$ = $` doesn't work - the dollar sign ends up undefined. 

However, `Ember.$ === jQuery`, so I was just using the global to avoid destructuring linting errors. 

I'm still trying to understand why `$ = $` isn't working...